guide/06-imagery-and-raster-analysis/lidar_basics_part1.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p style="user-select: auto;">We may have to edit the names and hyperlinks to other parts after we rename the rest of the files.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>